src/constants/value_commit_v.rs

```diff
@@ -0,0 +1,3735 @@
+            228, 165, 43, 250, 128, 26, 119, 104, 102, 214, 22, 205, 63, 39,
+        ],
+        [
+            199, 237, 20, 54, 148, 72, 88, 148, 123, 105, 25, 164, 11, 232, 235, 189, 96, 233, 62,
+            135, 91, 142, 235, 142, 234, 140, 14, 69, 117, 250, 166, 47,
+        ],
+        [
+            116, 35, 156, 87, 255, 89, 195, 168, 172, 105, 190, 192, 148, 130, 232, 13, 56, 119,
+            34, 194, 60, 217, 52, 133, 68, 43, 32, 153, 157, 71, 186, 26,
+        ],
+        [
+            46, 114, 149, 114, 124, 70, 191, 152, 198, 131, 215, 210, 50, 161, 225, 195, 54, 250,
+            65, 223, 230, 22, 42, 213, 218, 179, 84, 70, 83, 219, 187, 4,
+        ],
+    ],
+    [
+        [
+            57, 55, 194, 85, 109, 196, 8, 231, 26, 210, 212, 126, 1, 85, 185, 159, 8, 26, 209, 240,
+            11, 12, 20, 215, 246, 91, 252, 154, 53, 221, 39, 36,
+        ],
+        [
+            244, 140, 53, 249, 119, 142, 191, 244, 255, 115, 76, 19, 31, 183, 209, 83, 105, 104,
+            129, 119, 159, 48, 236, 249, 111, 161, 44, 85, 33, 43, 249, 0,
+        ],
+        [
+            166, 98, 12, 81, 242, 178, 26, 32, 45, 53, 147, 16, 59, 195, 217, 28, 155, 132, 210,
+            11, 21, 237, 216, 252, 122, 82, 122, 171, 47, 9, 18, 17,
+        ],
+        [
+            11, 197, 94, 238, 100, 217, 203, 50, 3, 28, 197, 23, 38, 84, 79, 245, 152, 99, 158,
+            173, 142, 54, 163, 0, 93, 215, 0, 65, 3, 227, 31, 24,
+        ],
+        [
+            51, 126, 228, 14, 133, 60, 90, 146, 177, 249, 18, 88, 160, 102, 185, 47, 43, 57, 102,
+            87, 153, 53, 37, 26, 54, 202, 168, 196, 249, 233, 110, 59,
+        ],
+        [
+            201, 238, 70, 90, 245, 235, 31, 114, 160, 2, 41, 46, 95, 186, 82, 67, 198, 108, 72,
+            219, 145, 161, 153, 207, 172, 139, 134, 48, 54, 99, 141, 6,
+        ],
+        [
+            253, 67, 63, 76, 233, 148, 229, 36, 161, 148, 242, 153, 47, 128, 164, 24, 224, 97, 110,
+            119, 68, 161, 233, 227, 30, 237, 84, 202, 66, 252, 141, 27,
+        ],
+        [
+            189, 69, 12, 183, 72, 18, 162, 221, 64, 214, 158, 223, 84, 174, 13, 7, 175, 216, 79, 9,
+            134, 7, 186, 190, 109, 204, 93, 211, 41, 236, 117, 40,
+        ],
+    ],
+];
+
+pub fn generator<C: CurveAffine>() -> ValueCommitV<C> {
+    ValueCommitV(OrchardFixedBase::<C>::new(
+        C::from_xy(
+            C::Base::from_bytes(&GENERATOR.0).unwrap(),
+            C::Base::from_bytes(&GENERATOR.1).unwrap(),
+        )
+        .unwrap(),
+    ))
+}
+
+#[cfg(test)]
+mod tests {
+    use super::super::{TestFixedBase, NUM_WINDOWS, NUM_WINDOWS_SHORT};
+    use super::*;
+    use group::Curve;
+    use halo2::{
+        arithmetic::{CurveAffine, CurveExt, FieldExt},
+        pasta::pallas,
+    };
+
+    #[test]
+    fn generator() {
+        let hasher = pallas::Point::hash_to_curve(PERSONALIZATION);
+        let point = hasher(b"-v");
```

Comment:
This is wrong. The value commitment base is GroupHash^P("z.cash:Orchard-cv", "v").

```suggestion
        let hasher = pallas::Point::hash_to_curve(VALUE_COMMITMENT_PERSONALIZATION);
        let point = hasher(b"v");
```